posthog/api/proxy_record.py

```diff
@@ -45,14 +50,34 @@ def list(self, request, *args, **kwargs):
     def create(self, request, *args, **kwargs):
         domain = request.data.get("domain")
         queryset = self.organization.proxy_records.order_by("-created_at")
-        queryset.create(
+        record = queryset.create(
             organization_id=self.organization.id,
             created_by=request.user,
             domain=domain,
             target_cname=generate_target_cname(self.organization.id, domain),
         )
-        serializer = self.get_serializer(queryset, many=True)
-        return Response(serializer.data)
+
+        temporal = sync_connect()
+        inputs = CreateHostedProxyInputs(
+            organization_id=record.organization_id,
+            proxy_record_id=record.id,
+            domain=record.domain,
+            target_cname=record.target_cname,
+        )
+        workflow_id = f"proxy-create-{inputs.proxy_record_id}"
+        asyncio.run(
+            temporal.start_workflow(
+                "create-proxy",
+                inputs,
+                id=workflow_id,
+                task_queue=BATCH_EXPORTS_TASK_QUEUE,
+            )
+        )
+
+        return Response(
+            {"success": True},
+            status=status.HTTP_200_OK,
+        )
```

Comment:
Does this break the frontend? Looks like we're setting the `proxyRecords` to the result of this endpoint which could cause an issue.

Reply:
ah i was wondering if it was intentional or not, i'll switch that back

Reply:
looks like it's ignoring the result and re-fetching actually:

```
createRecordSuccess: () => actions.loadRecords(),
```

Reply:
Feel free to leave it if you like. I was going to pick off the frontend tasks (e.g. pay gate) so maybe it's best if you make all your changes and I can double back to finish off